From 6e4788f68f207fc1956587d4915bbdd7ed8a1fe1 Mon Sep 17 00:00:00 2001
From: Kai Pastor <dg0yt@darc.de>
Date: Mon, 2 Nov 2015 00:35:43 +0100
Subject: [PATCH] android_install: Install Qt apps to QT_INSTALL_LIBS

Fixes failures on "make install" when building Qt for Android.
Makes it possible to build Qt tools for later deplyoment.

Task-number: QTBUG-47453
Task-number: QTBUG-45095
Task-number: QTBUG-44347
Task-number: QTBUG-38452
Change-Id: I3d5f2e7fb780d5514a4359a46f3ae80faac6d897
---
 mkspecs/features/android/android.prf | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/mkspecs/features/android/android.prf b/mkspecs/features/android/android.prf
index f428f7d..55a6ae7 100644
--- a/mkspecs/features/android/android.prf
+++ b/mkspecs/features/android/android.prf
@@ -3,8 +3,9 @@ contains(TEMPLATE, ".*app") {
         !contains(TARGET, ".so"): TARGET = lib$${TARGET}.so
         QMAKE_LFLAGS += -Wl,-soname,$$shell_quote($$TARGET)
 
-        android_install: {
-            target.path=/libs/$$ANDROID_TARGET_ARCH/
+        android_install {
+            QTDIR_build: target.path=$$[QT_INSTALL_LIBS]
+            else: target.path=/libs/$$ANDROID_TARGET_ARCH/
             INSTALLS *= target
         }
     }
-- 
1.9.1

